src/Instantiator.php

```diff
@@ -115,6 +115,21 @@ public static function forceGet(object $object, string $property)
         return self::accessibleProperty($object, $property)->getValue($object);
     }
 
+    public static function getFindCriteriaFromAttributes(array $attributes): array
+    {
+        $criteria = [];
+        foreach ($attributes as $key => $value) {
+            if (0 === \mb_strpos($key, 'optional:')) {
+                continue;
+            }
+
+            $normalizedKey = false !== \mb_strpos($key, ':') ? \mb_substr($key, \mb_strpos($key, ':') + 1) : $key;
```

Comment:
I'm not sure whether we should simply remove all colon prefixes, or if we should explicitly only remove the `force:` prefix. I think it's safe to say that no column name will ever have a colon in it, but I'm not sure about your idea on this.

Reply:
I think it's fine as is.